tikv-client/src/main/java/com/pingcap/tikv/operation/KVErrorHandler.java

```diff
@@ -248,6 +248,6 @@ public boolean handleRequestError(BackOffer backOffer, Exception e) {
         BackOffFunction.BackOffFuncType.BoTiKVRPC,
         new GrpcException(
             "send tikv request error: " + e.getMessage() + ", try next peer later", e));
-    return true;
+    return false;
```

Comment:
mind if we add a comment for this change?

Reply:
fixed